BUILD

```diff
@@ -25,7 +25,7 @@ package_group(
 # use by passing `--//:enable_yosys=0` to `bazel build` or `bazel test`
 string_flag(
     name = "enable_yosys",
-    build_setting_default = "1",
+    build_setting_default = "0",
 )
```

Comment:
is this just a (lack of) rebasing artifact or an intentional change?

Reply:
Pulled in from the cherry pick, will rever.